File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        for(int i = 0; i < arr.length; i++) {
            for(int j = i + 1; j < arr.length; i++) {
                if(arr[i] == arr[j]) {
                    return true;
                }
            }
        }
    }
    return false;
}

File: Data Structures & Algorithms/binary-search/submission-0.py
import math
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        first, last = 0, len(nums)

        for num in nums:
            mid = first + ((last - first) // 2)

            if nums[mid] >= target:
                last = mid
            elif nums[mid] < target:
                first = mid + 1
        return l if (l < len(nums) and nums[l] == target) else -1
        
        
